packages/world/src/WorldResourceId.sol

```diff
@@ -102,9 +99,9 @@ library WorldResourceIdInstance {
         abi.encodePacked(
           resourceType,
           ":",
-          resourceNamespace == ROOT_NAMESPACE ? ROOT_NAMESPACE_STRING : resourceNamespace,
+          resourceNamespace == bytes14("") ? "<root>" : WorldResourceIdLib.toTrimmedString(resourceNamespace),
           ":",
-          resourceName == ROOT_NAME ? ROOT_NAME_STRING : resourceName
+          resourceName == bytes16("") ? "<root>" : WorldResourceIdLib.toTrimmedString(resourceName)
```

Comment:
small nit: let's use the `ROOT_NAMESPACE` and `ROOT_NAME` constants (imported from `constants.sol`) here instead of the inline `bytes14`